assets/app/scripts/controllers/build.js

```diff
@@ -42,6 +42,11 @@ angular.module('openshiftConsole')
 
     var watches = [];
 
+    var setLogVars = function(build, $scope) {
```

Comment:
Yeah isn't really necessary, was just preference for giving the fn everything it needs as args.  I'll cut `$scope`.